homeassistant/components/sensor/zha.py

```diff
@@ -187,7 +187,7 @@ def state(self):
         if self._state is None:
             return None
 
-        return round(float(self._state))
+        return round(float(self._state) / 10, 1)
```

Comment:
we should apply `ac_power_multuplier` and `ac_power_divisor` attribute values and don't use the hardcoded values. for multiplier/divisor as those would differ from vendor to vendor.